docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1899,9 +1899,9 @@ The `dart:math` library is always imported `as math`.
 
 As a general rule, if a feature is entirely self-contained (not requiring low-level integration into the Flutter framework) and is not something with universal appeal, we would encourage that that feature be provided as a package.
 
-We try to be very conservative with what we put in the core framework, because there's a high cost to having anything there. We have to commit to supporting it for years to come, we have to document it, test it, create samples, we have to consider everyone's varied desires which they may have as they use the feature, we have to fix bugs. If there's design problems, we may not find out for a long time but then once we do we then have to figure out how to fix them without breaking people, or we have to migrate all our existing widgets to the new architecture, etc.
+We try to be conservative with what we add to the framework because we have to support it. We have to document it, test it, and create samples. We have to consider the many ways people may use it. We have to fix bugs. If there are design problems, we may not find out for a long time. Once we do, we have to figure out how to fix them without breaking downstream code, or we have to migrate all our existing widgets to the new design.
 
-Basically, code is expensive. So before we take it, if possible, we like to see if we can prove the code's value. By creating a package, we can see if people use the feature, how they like it, whether it would be useful for the framework, etc, without having to take on the costs.
+Basically, code is expensive. So before we take it, if possible, we like to see if we can prove the code's value. By creating a package, we can see if people use the feature, how they like it, and whether it would be useful for the framework, all without incurring support costs.
```

Comment:
Similarly, this changes nuances of meaning and isn't helpful.

Reply:
The added specification of support costs here ties this paragraph to the previous whose revised first sentence ends with `because we have to support it`. The tie-in brings closure to the argument.

Reply:
(I wish there were something I could add here to move the conversation forward, but I think both the original and the revised version are really good in their own right.)

Reply:
Same here.